### PR TITLE
Test for missing mandatory field error thrown during domain update via save domain api.

### PR DESCRIPTION
**Pull request by labkey-bpatel**

### CODE REVIEW

File: src/org/labkey/test/tests/issues/IssueDomainTest.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+@Category({Issues.class, DailyA.class, Data.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 20)
```
Comment (labkey-jeckels) [inline src/org/labkey/test/tests/issues/IssueDomainTest.java]: This looks like it should be a lot faster than 20 minutes. Suggest removing this annotation and letting it use the default.

File: src/org/labkey/test/tests/issues/IssueDomainTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    @Test
+    public void mandatoryFieldTest() throws Exception
+    {
+        log("Remove mandatory field 'NotifyList'");
+        GetDomainCommand getCmd = new GetDomainCommand(DOMAIN_NAME, ISSUES_NAME);
+        DomainResponse getDomainResponse = getCmd.execute(this.createDefaultConnection(false), getContainerPath());
+        List<Map<String, Object>> getDomainCols = getDomainResponse.getColumns();
+
+        int index = 0;
+        for (Map<String, Object> col : getDomainCols)
```
Comment (labkey-jeckels) [inline src/org/labkey/test/tests/issues/IssueDomainTest.java]: Using an Iterator and calling its .remove() method would be a little cleaner

Comment (labkey-bpatel) [reply src/org/labkey/test/tests/issues/IssueDomainTest.java]: Good point! done!

File: src/org/labkey/test/tests/issues/IssueDomainTest.java (hunk 1 above)
Comment (labkey-bpatel) [reply src/org/labkey/test/tests/issues/IssueDomainTest.java]: that was a copy paste, now changed to 2 mins. On my local machine total time is 1m 46 secs (which includes build time and test time), so 2 mins should be good.

**Pull request merged**